.github/workflows/drivers_ci.yml

```
@@ -187,10 +187,12 @@ jobs:
           command_timeout: 60m
           script: |
             sudo dnf install -y bpftool ca-certificates cmake make automake gcc gcc-c++ kernel-devel clang git pkg-config autoconf automake libbpf-devel
+            # Remove, if present, any libs clone created by a previous job run.
+            rm -rf libs || true
```

```suggestion
            rm -rf libs
```
`rm -rf` does not fail if folder does not exist